src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
```

Comment:
Do you think it might make the code clearer if you put some line break to separate out the different type of import statements?